generator/README.md

```diff
@@ -0,0 +1,34 @@
+# Software Engineering for Machine Learning Systems
+
+## Generator
+
+This directory contains code to generate synthetic creatinine blood test results, acute kidney injury events, and patient admit and discharge within a hospital. It makes no attempt to be clinically accurate, but beleivable enough to reason about when building a system.
+
+A random set of patients is selected to suffer acute kidney injury, and the blood test results for these patients are constructed such that the [NHS reference AKI algorithm](https://www.england.nhs.uk/akiprogramme/aki-algorithm/) performs well. Bias is deliberately added to enable the design of simple machine learning models with better performance.
+
+
+Patient ages and sexes are reasonably distributed, as are their blood result values. Event times and frequencies bear some resemblance to what you'd see in a reasonbly sized hospital.
+
+The generator can be run with:
+
+```
+pip3 install -r ../requirements.txt
+./generator.py --data=../data --history=90 --days=30 --output=.
+```
+* `--data` specifies the location of the census and name datasets used to generate the synthetic population.
+* `--history` specifies the number of days of patient history that should be generated, written as a CSV dataset of historical blood test results.
+* `--days` specific the number of hospital events that should be generated, written as an MLLP encoding set of HL7 messages.
+* `--output` specified the directory in which output files should be written.
+
+The generator produces these outputs:
+* `messages.mllp` an MLLP encoded sequence of events at the hospital over the given number of days. It includes:
+    * `ADT^A01` events when a patient is admitted. These messages include patient demographic data.
```

Comment:
i m being really nitty here but events when feels weird
maybe events for admitting a patient / events for recording blood test results etc

Reply:
I switched it to _an MLLP encoded sequence of HL7 message representing events_